followme:
  gesture_filtering_number: 3
  goal_threshold: 0.2
  search_const_ang_vel: 0.5
  safety_circle_radius: 1.5
  target_tf_name: "target_0"

  #### TO BE CHANGED BASED ON ROBOT ####
  map_tf_name: "map" # Name of the map tf
  camera_tf_name: "camera_depth_optical_frame" # Name of the camera depth optical frame tf
  robot_base_tf_name: "base_link" # name of the robot base link tf
  move_base_node_name: "/move_base" # name of the move_base node
  cmd_vel_topic_name: "/cmd_vel" # name of the robot command velocity topic IT WASNT /cmd_vel before, it had /blabla/cmd_vel
  # TODO add cmd_vel_topic to publish different to robot one
  ######################################

track_person:
  display_img: False
  display_fps: False

  #### TO BE CHANGED BASED ON ROBOT ####
  rgb_topic: "/camera/color/image_raw" # topic of the rgb image
  depth_topic: "/camera/depth/image_raw" # topic of the depth image
  camera_info_topic: "/camera/color/camera_info" # topic of the camera info 
  camera_depth_optical_frame: "camera_depth_optical_frame" # $(param followme/camera_tf_name)  # Name of the camera depth optical frame tf
  ######################################


# obstacle_avoidance:
#   filtered_vel_topic: "/cmd_vel"
#   laserscan_topic: "/base_scan"
#   obstacle_consideration_range: 1.0

#CONTROLS HOW THE REIDENTIFICATOR WORKS; 
reidentificator:
  #True - does a calibration at the start and saves it to calibration_path; False- goes to calibrate_path and instances that object instead;
  calibration_bool: True
  #THE FILE MUST BE A PKL FILE -> filename.pkl, they are stored in the calibrations folder
  calibration_filename: "calibration_david_guida_blue_shirt.pkl"

#Parameters for the person buffer class in track_person_node (proxemics)
person_buffer:
  buffer_size: 5
  base_distance: 0.4 #base distance added on the tolerance to check whose pos it is
  velocity_influence: 0.25 #the influence velocity has on the tolerance
  side_bias: 1.8 #the bias toward a side, bias of 1 is no bias.
  activity_treshold: 15 #seconds that the object will be deleted after no activity

general:
  topic_name: "/general_control"

state:
  topic_name: "/followme_state"